context/app/api/vitessce_confs/base_confs.py

```diff
@@ -395,6 +396,7 @@ def get_conf_cells(self):
                 "K-Means [Shape-Vectors]",
                 "K-Means [Texture]",
                 "K-Means [Total] Expression",
+                "K-Means [tSNE_All_Features]"
```

Comment:
I can't suggest an edit that spans all the rows, but can you define this list once, and then use if for both?

Reply:
Good call.